### ISSUE TITLE
printf generate duplicate outputs

### ISSUE THREAD

**Issue by thalesmaoa** [Calysto/octave_kernel#129]
Hi, for some reason my new installation is all bugged.
Besides of #128, when I try to use printf it generate the same result twice.

```
printf("Test")
``` 
```
Test
Test
```

**Comment by thalesmaoa** [comment]
Just found a workaround for #129, 

```
printf('Test\n')
```

```
Test
```

I installed jupyter and octave_kernel with pip3 -U flag. 

```
$ jupyter-notebook --version
5.4.1
```

About octave_kernel, I don't know how to get its version.

**Comment by dsblank** [comment]
The work around was to add a newline to the string to be printed? You can use pip to list or show:

```
pip show octave_kernel
OR
pip list
```

**Comment by thalesmaoa** [comment]
`The work around was to add a newline to the string to be printed?`

Yes, for some reason a newline character fix the double printing.

```
$ pip3 show octave_kernel
Name: octave-kernel
Version: 0.28.3
Summary: A Jupyter kernel for Octave.
Home-page: http://github.com/Calysto/octave_kernel
Author: Steven Silvester
Author-email: steven.silvester@ieee.org
License: BSD
Location: /usr/local/lib/python3.5/dist-packages
Requires: metakernel, ipykernel, jupyter-client
```

**Comment by syam9030** [comment]
We are unable to integrate packages in Octave kernel with jupyter notebook would you please help in issue.

**Comment by blink1073** [comment]
Hi @syam9030, can you please open a new issue describing what you have tried so far and what problem you are seeing?

**Comment by tharun001** [comment]
hey guys for some build packages dependencies  of octave kernel printf command is not working then you may use fprintf("TEST")

**Comment by blink1073** [comment]
Fixed by https://github.com/Calysto/metakernel/pull/179.
